lib/mangoex/client.ex

```diff
@@ -18,10 +18,18 @@ defmodule Mangoex.Client do
     GenServer.call(@client_name, {:create_user, body})
   end
 
+  def create_user(type, body) do
+    GenServer.call(@client_name, {:create_user, body, type})
```

Comment:
I think `type` and `body` need swapped around here (based on the order of the related `handle_call`.

Reply:
done :)